week6/sentimental-credit/credit.py

```diff
@@ -0,0 +1,46 @@
+while True:
+    card = int(input("Card: "))
+    if card > 0:
+        break
```

Comment:
Add try/catch similar to `cash.py`

![Screenshot 2022-08-25 at 16 31 00](https://user-images.githubusercontent.com/80166473/186692929-66bf01ed-1273-49b3-874f-9f1755553ad8.png)